report.py

```diff
@@ -131,6 +131,9 @@ def set_start_date(self, start_date):
     def set_end_date(self, end_date):
         self._data['end_date'] = end_date
 
+    def set_days(self, days):
```

Comment:
What is this for?

Reply:
I wanted the report json to include the number of days for the report (we have the start_date and end_date and could calculate the number of days from that, but this seemed easier and cleaner)

Reply:
Where is `self._data['days']` being used?